apps/explorer/config/test.exs

```diff
@@ -6,7 +6,10 @@ config :bcrypt_elixir, log_rounds: 4
 # Configure your database
 config :explorer, Explorer.Repo,
   database: "explorer_test",
-  hostname: "localhost",
+  hostname: System.get_env("PGHOST") || "localhost",
+  username: System.get_env("PGUSER") || "postgres",
+  password: System.get_env("PGPASSWORD") || "postgres",
```

Comment:
You are passing the same `postgres` as `PGUSER` and `PGPASSWORD` in the `docker-compose`. Do we need to pass the same values as the default ones for this variables?

Reply:
Do we need to? no.

That said, I prefer to do it for the sake of explicitness and self-documentation. If you see the env vars in the docker-compose you know they exist in case you need to change them.

Reply:
Okay, just wanted to be sure that this was intentional. I mean, more changes to the original code -> more chances of merge conflicts. I'll leave this up to you!